### DISCUSSION TITLE
Power Supplies?

### DISCUSSION THREAD

**Discussion by RK11111111111** [probonopd/MiniDexed#620] [General]
What do you all use for power supplies?  I had a surge protector with usb ports and I tried to plug the Minidexed on one Rpi and Mixxx on another.  For some reason the midi controller which has the DAC was making severe buzzing noises.  I couldn't figure it out, I unplugged Minidexed and the buzzine went away.  I plugged Minidexed into a Motorola usb charger, and the noise went away for Mixxx, and incidentally the Minidexed sounded much cleaner.  My suspicion is that the DAC, raspiaudio,  is very sensitive to noise.

However, I was jamming out last night with Mixxx to layer samples and both of the Raspberry pi's sort of froze at the same time.  I was able to restart everything, however it seems like these devices would do better with cleaner power sources.  I'm curious what people are using.

**Comment by plamikcho** [comment]
I am using batteries - 2x18650 with MT3608 converter plugged directly in the Pi 5V pin. No problems with noise.

**Comment by Banana71** [comment]
All of the following voltage sources work for me without any problems.  I've never noticed any differences in sound.  I use the GY PCM5102 DAC.  Original Raspberry Pi power supply: https://www.berrybase.de/offizielles-raspberry-pi-usb-c-netzteil-5-1v/3-0a-eu-schwarz

https://amzn.eu/d/9ZMGdFO

https://amzn.eu/d/6uK2y4t

**Reply by probonopd** [reply]
Second this, the Original Raspberry Pi power supplies never have failed me.

**Comment by diyelectromusic** [comment]
I use a range, but I can say from my Arduino experiments, some USB power supplies don't provide a useful (or good) ground, so it might be related to that?  Sometimes adding an extra ground connection can solve issues like that (providing it doesn't create ground loops).

**Comment by RK11111111111** [comment]
Using usb power supplies that plugged into the outlet rather than the power supplies built into the surge strip appeared to clear the problem.  My Rpi didn't come with power adapters, but I have plenty.  The one that came with a Rpi touch screen worked really well, and it was heavier than the others.
